sdk/identity/azure_identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- A `get_token()` error motivated by an HTTP response carries that response. See the [troubleshooting guide](https://aka.ms/azsdk/rust/identity/troubleshoot#find-relevant-information-in-errors) for example code showing how to access the response.
```

Comment:
"motivated". Perhaps "caused"? Even if that's an arcane but valid use - which I normally appreciate - I imagine users might be scratching their heads as well.